src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,101 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private Entry<K, V>[] table;
+    private int size;
+    private int threshold;
+    private final float loadFactor;
+
+    static class Entry<K, V> {
```

Comment:
Remember to parameterize the Node class. It should be parameterized with key and value types to prevent problems with data types.